dist/atheme.conf.example

```diff
@@ -50,6 +50,7 @@
  * Asuka 1.2.1 or later                         modules/protocol/asuka
  * Bahamut 1.8.x                                modules/protocol/bahamut
  * Charybdis IRCd                               modules/protocol/charybdis
+ * ChatIRCd                                     modules/protocol/chatircd1.1
```

Comment:
Who uses this? Is it really necessary in Atheme? A separately maintained version also has its benefits: independence from Atheme development and we're not the ones who need to fix the issues with it.

Reply:
I know of at least a handful of networks, including mine, that run ChatIRCD.